optaplanner-distribution/src/main/assembly/scripts/runExamples.bat

```diff
@@ -4,8 +4,7 @@ setLocal enableExtensions enableDelayedExpansion
 rem Most examples run (potentially slower) with max heap of 128 MB (so -Xmx128m), but 1 example's dataset requires 1.5 GB
 set "jvmOptions=-Xms256m -Xmx1536m -Dorg.optaplanner.examples.dataDir=sources/data/"
 set "mainClass=org.optaplanner.examples.app.OptaPlannerExamplesApp"
-set "mainClasspath="
-for %%i in (binaries\*.jar) do (set "mainClasspath=!mainClasspath!;%%i")
+set "mainClasspath=binaries/*;../binaries/*"
```

Comment:
The reason for `*` instead of `*.jar` is http://stackoverflow.com/a/1275467 . Looks like the `*.jar` does not work on Windows for whatever reason.

I've tried both powershell and the standard windows `cmd`. However, It would be definitely good to try on Win 7 or Win 8 as well.